executor/executor.go

```diff
@@ -1263,7 +1263,7 @@ func init() {
 			ctx = opentracing.ContextWithSpan(ctx, span1)
 		}
 
-		e := &executorBuilder{is: is, ctx: sctx}
+		e := newExecutorBuilder(sctx, is, nil, "")
```

Comment:
Why using an empty scope here?

Reply:
To keep the same configuration with the old code. But I think it's also fine to change it to `oracle.GlobalTxnScope`